chat-widget/src/components/startchaterrorpanestateful/common/defaultErrorPaneIconImageProps.ts

```diff
@@ -0,0 +1,12 @@
+import { IImageProps } from "@fluentui/react";
+import { ImageFit } from "@fluentui/react";
+import { AlertIcon } from "../../../assets/Icons";
+
+export const defaultErrorPaneIconImageStyleProps: IImageProps = {
```

Comment:
could you confirm look&feel is uniform in Android and iOS devices? using chrome, safari, edge

Reply:
Tested on few android / iOS devices, see below for some samples: https://microsoft-my.sharepoint.com/:f:/p/kevinlin/Eq6xXZf4pElGqWnyzt783BEBFwnRFOohp881DwptHnf9Lw?e=tcW8xG